cypress/e2e/patrimoine.cy.js

```diff
@@ -12,41 +12,54 @@ import "cypress-axe"
 context("Full simulation", () => {
   beforeEach(() => {
```

Comment:
```suggestion
  before(() => {
```
`beforeEach` n'est pas nécessaire si il n'y a qu'un seul test